curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/step-003.md

```diff
@@ -16,7 +16,7 @@ Inside the `head` element, nest a `meta` element with an attribute named `charse
 You should have a `meta` tag.
 
 ```js
-assert(code.match(/<meta\s.*>/i));
+assert(code.match(/<meta.*\/?>/g));
```

Comment:
```suggestion
assert(code.match(/<meta.*?\/?>/is));
```

Reply:
Incorporated as suggested. I brought the global flag over from step 18, which tested fine with and without self closing tags, however it was necessary due to the additional meta tag for that exercise.

Also appear that without the addition of the greedy operator, <meta> by itself might not have passed. Appears to be working in safari and chrome with changes as suggested.

Reply:
Thank you. The `g` flag is pointless here, because we are not trying to count how many there are - just that there is at least one match.

Reply:
Understood. I realized after submitting that it wasn't necessary. Thanks for working through it with me.